src/models/admin.jsx

```diff
@@ -71,20 +70,38 @@ const AdminContextProvider = ({ children }) => {
     });
   };
 
-  const approveUser = async (userId, user) => {
+  const approveUser = async (userId) => {
     // Approve an employer account, send to his email the password reset email as to notify him the approval
-    if (user.role === ROLES.EMPLOYER) {
-      await updateUser(userId, { status: USER_STATUS.APPROVED });
-      await auth.resetPassWordByEmail(user.email);
-    }
+    await runTransaction(database, async (transaction) => {
+      await transaction.update(doc(database, "Users", userId), {
+        status: USER_STATUS.APPROVED,
+      });
```

Comment:
there is no need to use runTransaction for a single operation. 

do we still need to send the email?

Reply:
I was under the impression that on thursday we decided we didn't? But I can't remember to well, will ask the group in slack

Reply:
OK ya that's correct Neha confirmed in slack